core/src/mongosqltranslate.rs

```diff
@@ -261,17 +284,23 @@ pub struct Namespace {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(transparent)]
 pub struct GetNamespacesCommandResponse {
+    #[serde(flatten)]
     pub namespaces: BTreeSet<Namespace>,
 }
 
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(transparent)]
 pub struct GetMongosqlTranslateVersionCommandResponse {
+    #[serde(flatten)]
     pub version: String,
 }
 
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(transparent)]
 pub struct CheckDriverVersionCommandResponse {
+    #[serde(flatten)]
```

Comment:
Changes so that these commands worked properly. All were throwing an error from the library